easybuild/framework/easyblock.py

```diff
@@ -3938,6 +4300,17 @@ def xs2str(xs):
         else:
             self.log.debug("Skipping RPATH sanity check")
 
+        if get_software_root('CUDA'):
+            if shutil.which('cuobjdump'):
+                cuda_fails = self.sanity_check_cuda()
+                if cuda_fails:
+                    self.log.warning("CUDA device code sanity check failed!")
+                    self.sanity_check_fail_msgs.extend(cuda_fails)
+            else:
+                raise EasyBuildError("Failed to execute CUDA sanity check: cuobjdump not found")
```

Comment:
That suggestion is not correct, I'm working on it

Reply:
I'm... confused. If `shutil.which('cuobjdump')` fails to return anything, that means we've failed to find `cuobjdump`, and thus we've failed to execute the sanity check, right? What am I missing here? :)